packages/lambda-xyz/src/cli/serve.ts

```diff
@@ -49,56 +49,82 @@ async function main(): Promise<void> {
     if (Env.get(Env.PublicUrlBase) == '') {
         process.env[Env.PublicUrlBase] = `http://localhost:${port}`;
     }
-    let projection: number = Epsg.Google.code;
-    const filePath = process.argv[2];
 
-    const tileSetName = 'local';
-    if (filePath != null) {
-        const tileSet = new TileSetLocal(tileSetName, Epsg.Google, filePath);
-        await tileSet.load();
-        const tiffFiles = tileSet.tiffs.map((c) => c.source.name).join(', ');
-        // TODO is there a better name for this
-        tileSet.setTitle(`Local - ${tiffFiles}`);
-        TileSets.set(tileSet.id, tileSet);
-        LogConfig.get().info({ tileSets: [...TileSets.keys()] }, 'LoadedTileSets');
-        projection = tileSet.projection.code;
-    }
+    if (process.argv[2] === '--use-dynamodb') {
```

Comment:
I don't really like the idea of having this large `if` here.

is there a way to use dynamodb by default and just override what is being returned by dynamodb when a folder is provided